### PR TITLE
Es6 modules

### PR DESCRIPTION
**Pull request by daniel-sungwon-lee**
image

### CODE REVIEW

File: es6-modules/src/components/todo-app.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      createElement('h1', { class: 'mb-4' }, 'JS Todo'),
+      this.todoForm.render(),
+      this.todoList.render(this.todos)
+    );
+    const { nextId, todos } = this;
+    this.onUpdate({ nextId, todos });
+  }
+}
+
+export = TodoApp;
```
Comment (scbowler) [inline es6-modules/src/components/todo-app.js]: Should be `export default TodoApp`

File: es6-modules/src/components/todo-form.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        ])
+      ])
+    );
+    this.element.addEventListener('change', this.handleChange);
+    this.element.addEventListener('submit', this.handleSubmit);
+    return this.element;
+  }
+}
+
+export = TodoForm;
```
Comment (scbowler) [inline es6-modules/src/components/todo-form.js]: Should be `export default TodoForm;`

File: es6-modules/src/components/todo-list.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      createElement('ul', { class: 'list-group shadow-sm mb-4' }, todos.map(todo => {
+        return this.renderTodoItem(todo);
+      }))
+    );
+    todoList.addEventListener('change', this.handleChange);
+    return todoList;
+  }
+}
+
+export = TodoList;
```
Comment (scbowler) [inline es6-modules/src/components/todo-list.js]: Should be `export default TodoList;`

File: es6-modules/src/lib/create-element.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  toArray(children).forEach(child => {
+    if (!(child instanceof HTMLElement)) {
+      child = document.createTextNode(child);
+    }
+    parent.appendChild(child);
+  });
+  return parent;
+}
+
+export = createElement;
```
Comment (scbowler) [inline es6-modules/src/lib/create-element.js]: Should be `export default createElement;`

File: es6-modules/src/lib/noop.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+function noop() {}
+
+export = noop;
```
Comment (scbowler) [inline es6-modules/src/lib/noop.js]: Should be `export default noop;`

File: es6-modules/src/lib/to-array.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+function toArray(value) {
+  return Array.isArray(value) ? value : [value];
+}
+
+export = toArray;
```
Comment (scbowler) [inline es6-modules/src/lib/to-array.js]: Should be `export default toArray;`

File: es6-modules/src/components/todo-app.js [hunk 7]
```diff
@@ -0,0 +1,56 @@
+import createElement from '../lib/create-element';
+import noop from '../lib/noop'
```
Comment (scbowler) [inline es6-modules/src/components/todo-app.js]: Here you can take advantage of the `index.js` file in the `lib` folder and import both of these on one line.

```JAVASCRIPT
import { createElement, noop } from '../lib'

**Review by scbowler — CHANGES_REQUESTED** [review]
Export statements need fixed, see comments for details.

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**